docs/guided-onboarding.md

```diff
@@ -1,18 +1,17 @@
 # Collect logs and metrics using the guided onboarding
-The guided onboarding in Kibana or in a serverless Observability project walks you through collecting logs and metrics using the Elastic Distribution for OpenTelemetry Collector.
-To configure the Elastic OTel Collector manually, refer to the [manual configuration](manual-configuration.md) docs.
+The guided onboarding in Kibana or in a serverless Observability project walks you through collecting logs and metrics using the Elastic Distribution of OpenTelemetry (EDOT) Collector.
+To configure the EDOT Collector manually, refer to the [manual configuration](manual-configuration.md) docs.
 
 ## Before you begin
-The onboarding has the following requirements and limitations:
+The onboarding has the following requirements:
 
 - The **Admin** role or higher is required to onboard system logs and metrics. To learn more, refer to [Assign user roles and privileges](https://www.elastic.co/docs/current/serverless/general/assign-user-roles).
 - Root privileges on the host are required to run the OpenTelemetry collector used in this quickstart.
-- The Elastic OTel Collector only works on Kubernetes, Linux, and MacOS systems.
-- Refer to [Elastic OpenTelemetry Collector limitations](collector-limitations.md) for known limitations when using the Elastic OTel Collector.
+- The EDOT Collector only works on Kubernetes, Linux, and MacOS systems.

```

Comment:
>The EDOT Collector only works on Kubernetes, Linux, and MacOS systems.

The collector works on more platforms but at this moment we are providing out of the box deployment and configurations for these platforms